<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow
{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(225, 240, 255, 255), stop:0.497512 rgba(221, 243, 228, 255), stop:1 rgba(255, 229, 230, 255));
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="excellentEmployeesTableView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>260</width>
      <height>170</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#excellentEmployeesTableView
{
	border-radius:12px;
	background-color: rgba(255, 255, 255,125);
}</string>
    </property>
   </widget>
   <widget class="QTableView" name="signInTableView">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>350</width>
      <height>220</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#signInTableView
{
	border-radius:12px;
	background-color: rgba(255, 255, 255,125);
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="fingerprintStatusLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>190</y>
      <width>260</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#fingerprintStatusLabel
{
	border-radius:12px;
	color: rgb(88, 71, 175);
	background-color: rgba(237, 233, 254,125);
	font: 57 15pt &quot;苹方UI-港&quot;;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="fingerprintButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>250</y>
      <width>260</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#fingerprintButton
{
	border-radius:12px;
	color: rgb(19, 123, 149);
	background-color: rgba(216, 243, 246,125);
	font: 57 15pt &quot;苹方UI-港&quot;;
}</string>
    </property>
    <property name="text">
     <string>录入指纹</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exportButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>250</y>
      <width>350</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#exportButton
{
	border-radius:12px;
	color: rgb(0, 145, 254);
	background-color: rgba(225, 240, 255,125);
	font: 57 15pt &quot;苹方UI-港&quot;;
}</string>
    </property>
    <property name="text">
     <string>导出为Execl</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>630</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
